alpha/apps/kaltura/lib/batch2/bcdl/kBusinessConvertDL.php

```diff
@@ -2,6 +2,25 @@
 
 class kBusinessConvertDL
 {
+
+
+private static function shouldDeleteOldAsset($oldAsset,$entry)
```

Comment:
This method is very specific to replacement flow, I would place the code inside replaceEntry method or at least name it to shouldDeleteMissingAssetDuringReplacement or something like it.